test/authlete/model/test_service.rb

```diff
@@ -142,6 +142,7 @@ class ServiceTest < Minitest::Test
   ATTRIBUTE_KEY                                 = '<attribute0-key>'
   ATTRIBUTE_VALUE                               = '<attribute0-value>'
   ATTRIBUTES                                    = [ Authlete::Model::Pair.new(key: ATTRIBUTE_KEY, value: ATTRIBUTE_VALUE) ]
+  SUPPORTED_CUSTOM_CLIENT_METADATA              = 'JSON string'
```

Comment:
Can you replace the `'JSON String'` with `[ '<supported-custom-client-metadata0>', '<supported-custom-client-metadata1>' ]` ?

NOTE: `supportedCustomClientMetadata` is a string array object, not a string object.

Reply:
I have made the change.